aws.go

```diff
@@ -261,12 +261,12 @@ func (a *awsAttributer) Logger() ctxLogger {
 }
 
 // httpAttributes returns a slice of slog.Attr for the http request and response
-func httpAttributes(r *http.Request, sw *statusWriter) []slog.Attr {
+func httpAttributes(r *http.Request, sw responseRecorder) []slog.Attr {
 	return []slog.Attr{
 		slog.String(awsHTTPMethodKey, r.Method),
 		slog.String(awsHTTPURLKey, r.URL.String()),
 		slog.Int(awsHTTPStatusCodeKey, sw.Status()),
-		slog.Int64(awsHTTPRespLengthKey, sw.length),
+		slog.Int64(awsHTTPRespLengthKey, sw.Length()),
```

Comment:
How did you know you needed to do this? Is it a computed length vs stored length?

Reply:
I see. `.length` wasn't exposed to that interface.